### ISSUE TITLE
Option for clean ADV textbox

### ISSUE THREAD

**Issue by Ascoat** [07th-mod/guide#7]
Could we have an option for a simple solid textbox?  ADV mode is nice, but the logo and trees are distracting.

You're welcome to the quick version I made for my own use.  I used an Umineko PS3 screenshot as a reference for opacity and edge feathering, and it'd be easy to add the rounded corners if I knew the text boundaries in ADV mode.

![windo_filter_adv](https://user-images.githubusercontent.com/20040718/31420642-fd818992-adff-11e7-8751-eb62190e4cfc.png)
![screenshot](https://user-images.githubusercontent.com/20040718/31420630-ead2ef66-adff-11e7-93d6-3172c43421bb.jpg)
![umineko](https://user-images.githubusercontent.com/20040718/31420737-8e3502c0-ae00-11e7-8bfc-9919ab354a14.jpg)

**Comment by ItaloKnox** [comment]
You should be able to tweak the transparency in game to make it less visible, but I'll see if I can include yours as an alternative.

**Comment by Ascoat** [comment]
Making the textbox transparent makes the text hard to read against light backgrounds, so it's not ideal.

If mine's good enough, then here's an updated version.  I used a more accurate method to estimate Umineko's opacity, which returned around 70% instead of 75% like my first one.

![screenshot2](https://user-images.githubusercontent.com/20040718/31421242-982a05a2-ae03-11e7-96b3-74b070e9d756.jpg)
![windo_filter_adv](https://user-images.githubusercontent.com/20040718/31421264-ae4993b6-ae03-11e7-9eee-27422bc94a0e.png)

**Comment by enumag** [comment]
@ItaloKnox I don't think we should add some more support for this. Those who want to replace it can do so after all. Perhaps just add a note to our wiki?

@Ascoat Rounded corners thing was the first thing @Grelo tried back when we added the ADV-mode. But it is a bit outdated in modern VNs so we used the current filter instead. To use a filter with rounded corners you would have to adjust the text positioning which you can do by changing the values in Update/init.txt.

**Comment by Ascoat** [comment]
I was thinking it'd just be an optional file on the wiki.  Maybe one day when there's a graphical installer it could have a page where you choose one, but it's not worth having multiple versions of the mod.

Why would you have to adjust the text positioning?  The side margin on the console versions' text boxes is wider than the corners, so it's still bounded in a rectangle.  (Higurashi Sui's boxes look very similar to Umineko's).

**Comment by enumag** [comment]
@Ascoat Because since I tried the rounded-corners variant previously I remember the positioning was quite different. As it is the text can go all the way near the bottom of the screen - it's rare but sometimes unavoidable. Since the box with rounded corners has a bottom margin the text would not fit with current positioning. Also the text does not need the padding on the sides which we have for the decorative branches.

**Comment by ItaloKnox** [comment]
@enumag Yeah, that's what I was thinking about. Sorry, I did step away from this issue but I didn't forget it. Just had some trouble with the company hosting the servers I use to push our patches, lost our entire configuration and they actually cleaned my backups without my consent. Took me some time but it should be fixed already.

If you guys are comfortable with the last version posted I can push it to the wiki.

**Comment by enumag** [comment]
@ItaloKnox Ouch. Why did they delete the data? I hope it's not copyright trouble. As for what version of the simple box we should put on the wiki that's really up to @Ascoat. I don't really care if we go with rounded corners or not.

**Comment by ItaloKnox** [comment]
I came here to open an issue for discussion and I found your comment. For some reason, I missed it, sorry about that.

So... I actually goofed and did a ``rm -rf`` in my user folder. Pretty dumb error but mostly because of the misfortune of the folder I was trying to delete in the root of my home folder started with ``shin`` and the same applied to my user folder. So I did a ``cd ..``, and let the autocomplete do his job. Big mistake, but shouldn't be an issue because I had scheduled backups twice a week. Or so I thought. The hosting company cleaned my backups and the backup job, couldn't really explain what happened and just said it was human error. *Deal with it, restore your off-site backups*. Sadly I didn't have any because the service has been always reliable to me. Lesson learned. 

**Anyway**, the server is a bit funky after that but nothing I can't fix. Since @Ascoat haven't said anything, I'll just go ahead and let this issue open for a while we discuss something else in a separate issue. After that, I'll put it somewhere.
